src/frozendict/core.py

```diff
@@ -50,11 +52,12 @@ def __new__(e4b37cdf_d78a_4632_bade_6f0579d8efac, *args, **kwargs):
             # empty singleton - start
             
             if self.__class__ == frozendict and not len(self):
-                try:
-                    self = cls.empty
+                global _empty_frozendict_singleton
+                if _empty_frozendict_singleton is None:
```

Comment:
I prefer to have the statements separated by a newline before. Can you add it?

Reply:
Happy to change this, but its not clear to me where you want the blank line?

Reply:
Before the if.